net-p2p/bitcoin-core/files/bitcoind.logrotate-r1

```diff
@@ -1,4 +1,4 @@
-/var/log/bitcoin/debug.log {
+/var/log/bitcoind/debug.log {
```

Comment:
This would require either a revision bump of *this* file or dropping all existing ebuilds that reference it. I decided to go with the later.

Reply:
Hmm. Can you explain your logic? This file was incorrect as it had existed. I took the opportunity to fix the bug in the existing version at the same time as adding the new version.

Reply:
> Hmm. Can you explain your logic?

Every change in the installed image requires a revision bump. bitcoin-core-25.0 image would change after this is applied.